posts/customer-360.md

```diff
@@ -0,0 +1,37 @@
+---
+
+# CUSTOMER 360
+## The 101 on the 360
+“Knowing Your Customer” is one of the most foundational, aspirational desires for any business–and yet the undertaking remains much more complex and nebulous than most people realize. As organizations grow, the sheer volume of data they have on their customers causes all sorts of challenges- from complex integration across lines of business, to messy data privacy policies causing headaches. Even the act of defining a customer can be a challenging process- Is it an individual? A household? A business? Someone who purchased a product 2 years ago? There’s a plethora of great articles out there for this more conceptual work; we may touch more on this in a later post, but in this blog we’re going to talk specifically about customer data and the technical aspects of tackling a Customer 360.
+
+So what is a “Customer 360”? The term gets thrown around a lot, but at its core it’s the ability to know everything relevant about a customer: their recent purchases, touchpoints with the organization, key attributes, contact information, preferences, etc. This is used for a variety of purposes such as cross-selling, targeted marketing, relationship profitability analysis, identifying VIP customers, and many other benefits to the organization. Further, there are plenty of benefits for the customers themselves- seamless customer experience, more targeted and relevant marketing, etc. More foundationally, it provides a clean single source of truth for a customer’s attributes.
+
+Now, where do we start, and what are the steps we must take to successfully implement this?
+
+![Customer 360 Visual](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/08/Customer360.png)
+
+## Data Discovery & Inventory
+First, we must identify the sources of customer data we’ll be relying on. Perhaps your company already manages this type of activity well: you have an enterprise data warehouse, good documentation, maybe even a robust data catalog. Likely, that is not the case. This first step involves developing an inventory of data sources and analyzing the complexity of obtaining that data. Profiling the data to understand inconsistencies will save time reconciling quality issues down the road and possibly even yield some short term wins by identifying data quality patterns that can quickly be resolved upstream.
+
+Within different departments and lines of business there will undoubtedly be “data gurus”- those people who really understand the data and its quirks. Identifying these people and developing a relationship with them is key to truly documenting and inventorying this data (fun side note- this could be a great jumping off point for a Data Catalog or data dictionary!). While the author of this blog is clearly biased; a third party consulting firm is most useful at this stage. The ability to see the forest from the trees and bring outside perspective provides a huge advantage.
```

Comment:
```suggestion
Within different departments and lines of business there will undoubtedly be “data gurus,” people who really understand the data and its quirks. Identifying these people and developing a relationship with them is key to truly documenting and inventorying this data (fun side note: this could be a great jumping-off point for a Data Catalog or Data Dictionary!). While the author of this blog is clearly biased, a third party consulting firm is most useful at this stage. The ability to see the forest from the trees and bring outside perspective provides a huge advantage.
```